config.go

```diff
@@ -142,6 +142,9 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Kubeless.Kubeconfig", "")
 	v.SetDefault("Kubeless.MinimumPriority", "")
 	v.SetDefault("Webui.URL", "")
+	v.SetDefault("Rabbitmq.MinimumPriority", "")
+	v.SetDefault("Rabbitmq.URL", "")
+	v.SetDefault("Rabbitmq.Queue", "")
```

Comment:
```suggestion
	v.SetDefault("Rabbitmq.URL", "")
	v.SetDefault("Rabbitmq.Queue", "")
	v.SetDefault("Rabbitmq.MinimumPriority", "")
```

For keeping same order than others